Tensile/SolutionStructs.py

```diff
@@ -2515,11 +2522,31 @@ def assignDerivedParameters(state):
       state["LdsPadB"] = 0 if state["ProblemType"]["TLUB"] else state["VectorWidth"]
       assert(state["LdsPadB"] >= 0)
 
+    if state["TransposeLDS"] == 1:
+      if not state["MatrixInstruction"]:
+        reject(state, "TransposeLds Supports only in MatrixInstruction=1")
+    if "MatrixInstruction" in state:
+      if state["TransposeLDS"] == 1:
+        if state["ProblemType"]["TLUA"] or state["ProblemType"]["TLUB"]:
+          reject(state, "TransposeLds requires TLU=0")
+    if state["TransposeLDS"] == 1:
+      if state["LdsBlockSizePerPad"] == -1:
+        state["LdsBlockSizePerPad"] = 256
```

Comment:
Always 256?

Reply:
not best optimized value. but value for adding padding..